packages/rum/src/domain/segmentCollection/startDeflateWorker.spec.ts

```diff
@@ -0,0 +1,178 @@
+import {
+  display,
+  isIE,
+  MonitoringMessage,
+  noop,
+  resetInternalMonitoring,
+  setDebugMode,
+  startFakeInternalMonitoring,
+} from '@datadog/browser-core'
+import { MockWorker } from '../../../test/utils'
+import { createDeflateWorker } from './deflateWorker'
+import { startDeflateWorker, resetDeflateWorkerState } from './startDeflateWorker'
+
+describe('startDeflateWorker', () => {
+  let deflateWorker: MockWorker
+  let createDeflateWorkerSpy: jasmine.Spy<typeof createDeflateWorker>
+
+  beforeEach(() => {
+    resetDeflateWorkerState()
+    deflateWorker = new MockWorker()
+    createDeflateWorkerSpy = jasmine.createSpy().and.callFake(() => deflateWorker)
```

Comment:
```suggestion
    createDeflateWorkerSpy = jasmine.createSpy('createDeflateWorkerSpy').and.callFake(() => deflateWorker)
```

nit: to ease debugging, it can be interesting to name the spies, especially when several are used in the same test